doc/docs.go

```diff
@@ -1,6 +1,6 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag at
-// 2020-05-20 14:44:28.92534 -0400 EDT m=+0.517600077
+// 2020-05-21 11:13:08.835806 +0800 CST m=+1.102151256
```

Comment:
Does this time update even if no swagger field is changed? This seems annoying...

Reply:
Yes, If we don't build swagger UI, this won't be updated.